File: fixtures/_badshop/api/otel_semconv.go
package main

import (
	"go.opentelemetry.io/otel/attribute"
)

func deprecatedHTTPAttrs() {
	_ = attribute.String("http.method", "GET")       // want: otel/semconv-drift
	_ = attribute.String("http.target", "/checkout") // want: otel/semconv-drift
	_ = attribute.String("http.request.method", "GET") // notwant: otel/semconv-drift
	_ = attribute.String("url.path", "/checkout")      // notwant: otel/semconv-drift
}

File: fixtures/_badshop/web/otel_semconv.ts
import { Span } from '@opentelemetry/api';

export function drift(span: Span) {
  span.setAttribute('http.method', 'GET'); // want: otel/semconv-drift
  span.setAttribute('http.target', '/checkout'); // want: otel/semconv-drift
  span.setAttribute('http.request.method', 'GET'); // notwant: otel/semconv-drift
  span.setAttribute('url.path', '/checkout'); // notwant: otel/semconv-drift
}
